dataaccess-couch.js

```diff
@@ -185,8 +191,7 @@ function init(dbname, options) {
         debug('Database is ready');
     }).catch(function(error) {
         console.error('Failed to initialise database(s)');
-        console.error(error);
-        process.exit(1);
+        throw new Error(error);
```

Comment:
Throwing within a `promise.catch()` block does not exit the process. I think `process.exit` is the only option here.